RockLib.Messaging.Kafka/KafkaReceiverMessage.cs

```diff
@@ -110,14 +126,16 @@ private static bool TryGetSchemaId(byte[] payload, bool containsSchemaId, out in
         protected override Task AcknowledgeMessageAsync(CancellationToken cancellationToken) => StoreOffsetAsync();
 
         /// <inheritdoc />
-        protected override Task RollbackMessageAsync(CancellationToken cancellationToken) => Tasks.CompletedTask;
+        protected override Task RollbackMessageAsync(CancellationToken cancellationToken) => Task.CompletedTask;
```

Comment:
Is this supposed to actually do some sort of rollback?  Not sure how CompletedTask would do that or am I missing something?  What was Tasks.CompletedTask?

Reply:
You don't have to (apparently). The base class has an implementation for this, and by returning a completed `Task`, you're basically ignoring all that.

`Tasks.CompletedTask` was a holdover. I believe when .NET created the `Task` type, there was no `CompletedTask`, so people created a shared `Task.FromResult(0)`, and that's not necessary anymore.

Reply:
Ah gotcha.  Ah I see that Tasks was deleted.